### ISSUE TITLE
Letting go of jQuery for HTML5 Developers [moved]

### ISSUE THREAD

**Issue by ebidel** [html5rocks/www.html5rocks.com#14]
This is [Issue 233](http://code.google.com/p/html5rocks/issues/detail?id=233) moved from the [old Google Code project](http://code.google.com/p/html5rocks/) and added by 2010-10-14T16:30:21.000Z by [paulir...@google.com](http://code.google.com/u/104337450658946893583/). Please review that bug for more context and additional comments, but update this bug.

Original labels: Type-Feature, Priority-P1, Tutorial, Milestone-7.1, New
### Original description

```
 - insertadjacenthtml, delegation, traversal

basically all the newish DOM methods (and array extras) that facilitate a jquery-ish style of dev.

getters and setters might come into play too...
```
